snappy/hwaccess.go

```diff
@@ -53,6 +53,18 @@ func validDevice(device string) bool {
 	return false
 }
 
+func validDeviceForUdev(device string) bool {
+	inValidPrefixes := []string{"/sys/devices", "/sys/class/gpio"}
```

Comment:
"inValid" or "invalid"? Those read like opposites.

Also, why are these invalid?  Would it make sense to have a whitelist rather than a blacklist, or something that is based on dynamic metadata?

It's probably okay to apply this for fixing a bug, given capabilities are coming soon anyway. Otherwise, I'd be concerned that we'll be expanding this list bug report after bug report.